cmd/query/Dockerfile

```diff
@@ -8,4 +8,5 @@ COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certifica
 EXPOSE 16686
 COPY query-linux /go/bin/
 ADD jaeger-ui-build /go/jaeger-ui/
-CMD ["/go/bin/query-linux", "--query.static-files=/go/jaeger-ui/"]
+ENTRYPOINT ["/go/bin/query-linux"]
+CMD ["--query.static-files=/go/jaeger-ui/"]
```

Comment:
To answer my own question: It seems that we need to use the `--entrypoint` option to override the entry point while anything after the image in the docker command overrides `cmd`

Reply:
I was discussing with @yurishkuro about this, and agree with him in that moving "--query.static-files=/go/jaeger-ui" as an argument to the `ENTRYPOINT` command is better. 

This is because overriding `CMD` in `docker run` overrides all the command parameters forcing users to pass in  "--query.static-files=/go/jaeger-ui/". 

By putting this in the `ENTRYPOINT`,  other overrides become easier, and people may still override the static file location by using the `-d` option

Reply:
That sounds reasonable to me, however in the case that a user did override `query.static-files`, the resulting command would look like:

`/go/bin/query-linux --query.static-files=/go/jaeger-ui/ --query.static-files=/my/desired/path`

which seems a little weird at first glance, but it works because `flags` sets the last value it encounters. Are we ok relying on this?

Also, it seems like the same change would make sense for the defaults in `cmd/standalone/Dockerfile`.

I'm very new to the Jaeger repo, so thank you for the patience here!